modules/nf-core/genmap/map/main.nf

```diff
@@ -0,0 +1,66 @@
+process GENMAP_MAP {
+    tag "$meta.id"
+    label 'process_high'
+
+    conda "bioconda::genmap=1.3.0"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/genmap:1.3.0--h1b792b2_1' :
+        'biocontainers/genmap:1.3.0--h1b792b2_1' }"
+
+    input:
+    tuple val(meta), path(index)
+    tuple val(meta2), path(regions)
+
+    output:
+    tuple val(meta), path("*.wig")      , optional:true, emit: wig
+    tuple val(meta), path("*.bedgraph") , optional:true, emit: bedgraph
+    tuple val(meta), path("*.txt")      , optional:true, emit: txt
+    tuple val(meta), path("*.csv")      , optional:true, emit: csv
+    path "versions.yml"                 , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "$meta.id"
```

Comment:
Will it better to also including meta2 info?
```suggestion
    def prefix = task.ext.prefix ?: meta2.id? "${meta.id}_${meta2.id}" : "$meta.id"
```

Reply:
Not entirely because the second input is optional :)